core/org.openjdk.jmc.flightrecorder.rules.jdk/src/main/resources/org/openjdk/jmc/flightrecorder/rules/jdk/messages/internal/messages_ja.properties

```diff
@@ -1,5 +1,5 @@
 #
-#  Copyright (c) 2018, 2019, Oracle and/or its affiliates. All rights reserved.
+#  Copyright (c) 2018, 2019, 2020, 2021 Oracle and/or its affiliates. All rights reserved.
```

Comment:
Only the first and the last year in the range should be in the copyright header. In this case the text should be:
Copyright (c) 2018, 2021, Oracle and/or its affiliates. All rights reserved.

Note the comma after the end year as well.

Reply:
I fixed it in 48d36d496f8085da84b434a6842da6009b72bfe1